harvester/DBInterface.py

```diff
@@ -113,6 +116,9 @@ def getConnection(self):
 	def getRow(self):
 		return self.dblayer.Row
 
+	def getType(self):
```

Comment:
This isn't called anywhere right now, right?

Reply:
Yes, it's used in Exporter.py for example line 114 (after this commit)

https://github.com/axfelix/globus_oai/pull/49/commits/205e36f80a22c574268d3b91f78cc2843a9f6e41

Reply:
Sorry, brain fart :)